packages/spec/integration/graphql-no-ssr/mock-schema.js

```diff
@@ -0,0 +1,34 @@
+import { makeExecutableSchema } from '@graphql-tools/schema';
+
+const typeDefs = `
+  type Subject {
+    id: ID!
+    title: String
+  }
+
+  type Query {
+    subject: Subject
+  }
+`;
+
+const values = Array.from({ length: 10 }, () => ({
+  id: 'def.456',
+  title: 'Server',
+})).concat({
+  id: 'abc.123',
+  title: 'Client',
+});
+
+const resolvers = {
+  Query: {
+    subject: () => values.pop(),
```

Comment:
How about something that is easier to understand and also always returns something without an error? 🙂

```suggestion
    subject: () => {
      id: 'abc.123',
      title: `Fetch no. #${++number}`,
    },
```